chore: remove comment

helenosheaa opened the pull request:
remove comment line left in plugin description

alamb reviewed (approved):
Looks like an improvement to me. Thanks @helenosheaa  I had noticed this when checking out the page

While checking out https://grafana.com/grafana/plugins/influxdata-flightsql-datasource/ I noticed the links also aren't rendered as links

![Screenshot 2023-04-24 at 1 07 31 PM](https://user-images.githubusercontent.com/490673/234067570-b5b92d35-6da3-47ed-81c5-065bd245ba81.png)

Maybe we can fix that too?

helenosheaa commented:
good shout @alamb , i've added them as links so they will all be under Resources on the lefthand side now, I've also updated the markdown to render proper links

alamb commented on src/README.md:
```diff
@@ -13,6 +11,8 @@ Select FlightSQL from the Type dropdown.
 Find out more here:
 
-https://github.com/influxdata/grafana-flightsql-datasource#grafana-flight-sql-datasource
+[Github](https://github.com/influxdata/grafana-flightsql-datasource#grafana-flight-sql-datasource)
```
This is how these render on github: https://github.com/influxdata/grafana-flightsql-datasource/tree/chore/remove-comment/src

![Screenshot 2023-04-24 at 3 51 47 PM](https://user-images.githubusercontent.com/490673/234101295-72d7fc9c-f7e8-42ba-bd9c-e09fec43b500.png)


Which seems like a nice improvement to me 👍
